_includes/assets/sass/_variables.scss

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+VARIABLES
+========================================================================== */
+
+// Colors
+$black: #000;
+$white: #ffffff;
+$yellow: #FFDB03;
+$pale-yellow: rgba(252, 247, 215, 0.63);
+$red: #CE132D;
+$grey-background: #E5E5E5;
+$grey-support: #B3BABF;
+
+$body-text: #323E48;
+
+// Custom media queries
+$xs-mobile-down: 340px;
+$sm-mobile: 375px;
+$md-mobile: 480px;
+$mobile: 576px;
+$tablet: 768px;
+$desktop: 992px;
+$lg-desktop: 1200px;
+$xl-desktop: 1440px;
+
+// Fonts
+$primary-font: 'Nunito Sans', sans-serif;
+$secondary-font: 'Source Sans Pro', sans-serif;
+$headings-font: 'Nunito Sans', sans-serif;
+
+// Font weight
+$base-font-weight-bold: 700;
+$base-font-weight-regular: 400;
+$headings-font-weight: 700;
+
+// Font sizes
+$base-font-size: 16px;
```

Comment:
I think the best practice here according to gov.uk is to use rem units with a px fallback? It looks like govuk-frontend uses a function to do this easily. https://github.com/alphagov/govuk-frontend/tree/899264cd89559d31f0a790682254d94ecf66418b/package/govuk/tools

Reply:
Yep, I will be using rems for font sizes etc.